ocaml-eglot.el

```diff
@@ -430,35 +430,46 @@ of result (LIMIT)."
       "local"
     "none"))
 
+(defun ocaml-eglot--construct-with-hole (arg hole)
+  "Construct over the current HOLE.
+It use the ARG to use local values or not."
+  (let* ((with-local-value (ocaml-eglot--construct-local-values arg))
+         (hole-start (cl-getf hole :start))
+         (result (ocaml-eglot-req--construct hole-start 1 with-local-value))
+         (range (cl-getf result :position))
+         (suggestions (append (cl-getf result :result) nil)))
+    (when (= (length suggestions) 0)
```

Comment:
I am not an elsip expert, but (as a micro optimization) would it make sense for this to be

```suggestion
    (when (null suggestions)
```

to avoid having to count all the suggestions?

Reply:
(I have created a new issue for that kind of improvement #25)